west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 240c7b9a691f6bcee824d1b26a40a368a916bb05
+      revision: pull/790/head
```

Comment:
Shouldn't we have this in an earlier commit? Will the other commits in this PR build without this change?